lib/search_config.rb

```diff
@@ -102,7 +101,7 @@ def searcher
 
   def combined_index_schema
     @combined_index_schema ||= CombinedIndexSchema.new(
-      content_index_names,
+      all_index_names,
```

Comment:
Why this change?

Reply:
this all worked fine previously as govuk used the same field definitions as mainstream, however now that the mainstream schema has been removed this breaks in fun ways - like filter terms are removed as they lived on the mainstream schema. :)

Reply:
I was confused by the definition of `all_index_names`:

```
  def all_index_names
    # this is used to process data in the rake file when `all` is passed in as previous we skipped `govuk`
    # we can't update index_names at this stage as it is used in multiple spots including the index filtering
    content_index_names + auxiliary_index_names + [govuk_index_name]
  end
```

Is it ok to include `auxiliary_index_names` here and in the test helpers?

Reply:
umm.. no that is a good point.

I thought that was content + govuk only.. let me correct that